packages/core-data/src/entities.js

```diff
@@ -63,6 +64,7 @@ export const rootEntitiesConfig = [
 		name: 'site',
 		kind: 'root',
 		baseURL: '/wp/v2/settings',
+		plural: 'sites',
```

Comment:
Same here it seems this entity doesn't really need plural.

Reply:
Needed for BC as well.